Firestore/Source/API/FSTUserDataConverter.m

```diff
@@ -394,7 +403,9 @@ - (nullable FSTFieldValue *)parseData:(id)input context:(FSTParseContext *)conte
       [result addObject:parsedEntry];
     }];
     // We don't support field mask paths more granular than the top-level array.
```

Comment:
Maybe:

// If context.path is nil we are already inside an array and we don't support field mask paths more granular than the top-level array.

Reply:
Done.